support-payment-api/src/main/scala/services/ContributionsStoreService.scala

```diff
@@ -52,6 +53,9 @@ object ContributionsStoreQueueService {
     private implicit val messageEncoder = Encoder[Message] { message =>
       import io.circe.generic.auto._
 
+      // copied from earlier version of circe-core to prevent seconds appearing in the serialised form
+      implicit val encodeLocalDateTime: Encoder[LocalDateTime] = (a: LocalDateTime) => Json.fromString(a.toString)
```

Comment:
one of the tests was failing due to the last part of the seriased form being `00:00:00` rather than the expected `00:00` so I had to add the old copy and pasted impelmentation.